src/cap-description.adoc

```diff
@@ -542,6 +545,7 @@ expanded base is 0 and top is 2^MXLEN^.
 | Tag      | zero  | Capability is not valid
 | SDP      | zeros | Grants no permissions
 | AP       | zeros | Grants no permissions
+| M (MXLEN=64) | zero | <<m_bit>>, encoded separately to AP for MXLEN=64
```

Comment:
I think this is not right. This chapter is about the capability encoding for zcheripurecap, so technically this field does not exist/has not been defined yet and is reserved